rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -81,6 +90,19 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
         ctx.close();
     }
 
+    private void validateRequestContent(ByteBufInputStream is) throws IOException {
+        try {
+            final ReadContext readContext = ReadContext.getReadContext(is, mapper);
+            final JsonNode jsonNode = readContext.nextValue();
+
+            if (jsonNode.get("id") == null) {
+                throw new IllegalArgumentException("missing request id");
+            }
+        } catch (IOException e) {
+             throw e;
```

Comment:
do we need this?

Reply:
yeah, because of:

```
final ReadContext readContext = ReadContext.getReadContext(is, mapper);
final JsonNode jsonNode = readContext.nextValue();
```

Reply:
Addressed